src/api.rs

```diff
@@ -289,4 +290,34 @@ mod tests {
             ..Default::default()
         };
     }
+
+    #[test]
+    fn test_endpoint_has_single_slash() {
```

Comment:
Might suggest to update the test name to be slightly clearer about what we're testing 💁🏼

```suggestion
    fn test_endpoints_handle_trailing_slashes() {
```